src/gui/activitywidget.cpp

```diff
@@ -313,24 +253,23 @@ void ActivityWidget::slotBuildNotificationDisplay(const ActivityList &list)
 
     const auto accId = list.first().uuid();
     QList<Activity::Identifier> strayCats;
-    for (const auto &id : _widgetForNotifId.keys()) {
-        NotificationWidget *widget = _widgetForNotifId[id];
+    for (auto it = _widgetForNotifId.cbegin(); it != _widgetForNotifId.cend(); ++it) {
```

Comment:
Is this change somehow related to the topic of the PR? I don't see the relation between using an iterator here and the new context menu.